libraries/lib-cloud-audiocom/sync/WavPackCompressor.cpp

```diff
@@ -198,7 +198,11 @@ struct Importer final
    bool IsValid() const
    {
       return Context != nullptr && SamplesCount > 0 &&
-             Format != undefinedSample && BlockId >= 0;
+             Format != undefinedSample &&
+             //The negative BlockId means silenced blocks and it's absolute value encodes it's length
+             //Older projects used to store them on the cloud as regular blocks.
+             ((BlockId >= 0 && SamplesCount > 0) ||
```

Comment:
`SamplesCount > 0` is already checked line 200.

Reply:
Thanks!